README.md

```diff
@@ -85,11 +85,17 @@ ssh-keyscan -t ssh-rsa github.com >> ~/.ssh/known_hosts
 
 Plugin version | GitBucket version
 :--------------|:-----------------
+1.2.x -        | 4.34.x -
 1.1.x -        | 4.32.x -
 1.0.x -        | 4.20.x -
 
 ## Release Notes
 
+### 1.2.0
+
+- Bump to GItBucket 4.32.0.
+- Pull request Hook support.
```

Comment:
```suggestion
- Bump to GitBucket 4.34.0.
- Pull request hook support.
```

Reply:
Could you fix casing for 1.1.0, too? "GItBucket" -> "GitBucket"

Reply:
updated too for 1.1.0.